src/main.js

```diff
@@ -324,7 +337,10 @@ function createDefaultArgvConfigSync(argvConfigPath) {
 			'',
 			'	// Enabled by default by VS Code to resolve color issues in the renderer',
 			'	// See https://github.com/Microsoft/vscode/issues/51791 for details',
-			'	"disable-color-correct-rendering": true'
+			'	"disable-color-correct-rendering": true,',
+			'',
+			'	// Enabled by default to record and send crash reports to Appcenter from all processes',
```

Comment:
Any particular reason you mention this explicitly here? Since the default is enabled, I wonder why it is called out, for privacy reasons?

Reply:
Just trying to be explicit, can remove if not required.

Reply:
Yes I think this is not needed. The other entry makes sense because we explicitly change a default of Electron imho.